SpectralCubeReprojectMaskExample_v3.ipynb

```diff
@@ -0,0 +1,1546 @@
qHgjoGA0T7nri0E1l/kjpTJxpvhGqPhgpEA7g4oPd+6apa/QppUGPFGvEVkhMD5BK4RkL4eLJcYqCUf7ZrEIBZMy1S7zkWMhH23/yaoKS2xPZMODfRa8U0Tkb8mLSpzpvN8L/E63/SyXilbIw0l/jNw/XEQkA35LVX91lWN3bAEGSkcJpUqsTU43PrHjFsRONT/dKYhhMRKyt3gLA8E+5oLzXKjfe7FifY1hDldhNEyYSzweagl3n57luP08zfj8JXZ0Y8pd5SpZKKlW8UfZFx3len+UmweUsUqLO254EGsN7VaRvfUaj0yO4knI4QrUk5AX7T3DkX2nqPTPc+7JPfzL8WfwifMFPtR4O7DqHEvbDmVTeaeMLQh4h3PA9d02Xm4m/n8DLwRuV9VjACJyBHiHiPykqv7mSkfr2Dr4EjKdPLGjBHxh0fbyIJti4RADhUNE2RwiAQV/NzV/D6EWGShkFD3LTOIRW6UhuXAvtF9p+bfFwj9QvpXB5GtoZyGesZw7P0qp2GZqvo+Hp4epJwG+QDVICIzyxFw//aUGn3v8KI/Xq/zpmTqfb7+PP+HtPfwrbQ+6TEV7LfiYiPwj8D7y+8t3AB/ttvFyIv49wMtV9WIOUFV9XEReD3wQcCK+TZmLTtHypndcwiTtBBQtCLCRYi7sJEx30tWGwV4qwSiBFBm1A3xuysOqx9mkxRPeMQ7bcYxnmCqVKRYOYcTHsrpc6FE2R+KlzCVFzraKeFOjzMUFzkcBmRUSFYYLKQaYjHwm2gH3zxxlJoaHWnN8pvmeXv1pth2bRcNV9cdE5JuBr+3sultV/7Lb9suJeLBYwBed9IKsJXenY9OzmsLH2wXt5OUXCfG9PgAym5DZFpmdJQz2UZQ+ajrIlNRpxDGJpFQp8S21m3no/2/vzqMku6vDjn/vW2rvvWfR7KNltCCJkRiMsEAIxElA6GBisEFYCsoxdhwLkIIBoyRgSOCcgG2Mgh0TBSIwKEhGdhxDYmwiEBJi1TKMRhpJSJrR7Jqe3rtrf+/mj/e6VdPTS3V3TXdX1/3ovDNVr+q9d6u6+urXv/r97m+kihuup5Mu+pLHGQ6OM1SYXx2UCcmJ6yuMVFxODrZxNC/8uPw8SVKsD3u4qD1JpuRxJA/FQBmtVjksfbwQPtaYN2QVWoHjxB8FRhcyI36uJD7b3NzWnrdrWkBIqNWoJV0zDDPhn4VGg/xIa5ILM21kPPAE+orQ4bm04ZL1UhzIJznmZCklRuZdu1skyabca2kLk+SrcGDMYXfpOAXJc7TyODl/HQU3DyPRMFBHhOMMMOz0M1g5RLlqVTFmswJGpwCLnxE/VxJ/uYhM14QQIDWPOI1pGqoVHCcXbZKY5vEqZc1TdMZ50S0S5pXOeHm3rOuScoWsG40s6fGTlCqdjPpbCcLyvCZKJbw1rA02gEA5VA6Wx/hl5aHJseapxBbchM8R56UYxxgChUL5OI5zeuwmssK+2FzUjPi5lmdzFxebMc0pDMfwvTVUg6FokeWa5FsNRhgtH6PsjeGIzyCHaHPWktIsPdUeusI0qcChHAqOgItLihw9yXMYcbN1tchFfDrTL6MqAUUNGC2X6Hf6T5ksVK4OMOokCLwKWemiqGO4+ARUyCY3UFnC8fzNSFdKp/giZ8TPp4qhMS1DxMP31pBObMQRn/GahYuntqQdJ0PBHUDE4bnqSdAQx8mSTaznQnkVGRJcwBbGwipD7iYOpude21JwGa+8yNMygOskSLtdZKSLV2T+JcecF3ixsGdyFIuIRzV9MTf3vpZzcgV603kGiufz4Ikc7ekLGSnsa+ybs0o0amWfBljUjHhL4sbEHCeH66RpT25hU+5q2rSLEkVCCRkrPj3jca6TjYpXVU5MDscU8ShW03Ql06xLengODJVdnAq86OTqiicMy1TCPhzJ4iWTqNNBVQLKmj9l0o7n9tDpbuTCjjEu2XCIrq4h+vt76C+dw4bCRYt7U1apqBTtckcx6bQZ8ar63+s92JK4MTHXiVah75UtdAVddEiKca0wImOzLhIR1TqvTJbLjc7VQU/yHDp9l6QLjkAQz8isZ0GG2rH5rtNBoFUqWuS47Ge4eOCU5ya9ThKaoBy4VCo+5VKCJ45v4B+PwYHKzxf0XrSCFdSd8n5VvR2YTNwicku8b04LG7xqzCrku220+xvYEK7l3FQbW7M+GxJpRpxBXGfmLpBq0E+5coyEt55M8uzJ9Uk3hWeTdIRAoa+ovFjN84L7AgPz6N5wJIXrZvCcJA4uJwv7Tqvpni8fYoCjHMqnODjYw7NHNtNfSvK08zzF8sEFvx+rmxDWsdV1JpFOEblXRJ4SkX0i8up5BvOeafbdVO/B1hI3JtabOJdNwWY6vQRJN6qF0u4Lm8qbedF9as6l0kqVw7huFym/h43uRbRriqPFCiNaZMgZZsh5kZP5ffOqi+44WcKwTKEyQJ6+aWufTNS6eXTgV9gz2E3GE54pjHKw+MMFvQ+tQDUae98gtwPfUdV3iEgCqKvMp4hcD7wb2C4if1/zUBtQ96K2LZfERSQFPAAkiV7/var6RyKyHbibaKzmo8CN8TfGnwAOAFcDX1HV+5chbLMEtgRbOTedYWNGyflVVOHJYY8CJTJez2QBq9mEYZ4O71LWhz24IuxznmWg8gKF0okFFZ9SQgSHSjCAzrKkWhAOsyb3KtLSQbmUp6/0BJ7bNu/rtZJG9ImLSDvRTMubADT6IdU7h+ZHwDGgF/jTmv2jwJ56Y2i5JA6UgDeo6lg86/SHIvIPwAeBP1PVu0Xki0RfNFjFoBayzk/RlYSkG3Jx1yClqkdIO4PlNobYQMXPR2VoJUWo49O2ypP+OtroJkAZ0wJVLVGsnCQMFzbxZj6J3+qEz0+dObxXRB6uuX+Hqt5Rc/9soA+4U0ReDjwC3KKq43NeX/UF4AVgvt0vp2i5PnGNTPw968ebAm8A7o33fxV4W3x7DCgAw9gs1VXL99bhi+A7iifKSDlJ
```

Comment:
These are both pretty cool to see the projected structures. The axis labels + units are probably quite important since I think fewer people are familiar with interpreting pv-slices.

Reply:
Referring to plotting of pv-slices.